pkg/operator/ceph/cluster/cluster.go

```diff
@@ -92,6 +92,13 @@ func (c *cluster) reconcileCephDaemons(rookImage string, cephVersion cephver.Cep
 	}
 	c.ClusterInfo.SetName(c.namespacedName.Name)
 
+	// Execute actions before the monitors are up and running
```

Comment:
This is unclear, how can we run commands against the mon store if the mons are not up and running? :)
So this should happen before the mons are updated to 16.2.8, right?

Reply:
These actions are only intended for upgrade or at least an existing cluster, i'll clarify the comment.